### PR TITLE
[T4A3][T13-B2] Gao Yu

### PR DESCRIPTION
**Pull request by GaoYu-Karen**

### CODE REVIEW

**Comment by GaoYu-Karen** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
```
Comment (KulaniT) [inline src/seedu/addressbook/data/person/Contact.java:2]: Header comments?

**Comment by KulaniT** [comment]
@GaoYu-Karen : Here you have to show how the inheritance can be utilized to reuse code. So think how can you use the Contact class as the parent class and reduce code at child classes (Phone, Email and Address).

**Comment by KulaniT** [comment]
@GaoYu-Karen :Some comments added. Please close the PR after reading comments

**Pull request closed without merging**